spec/models/price_policy_spec.rb

```diff
@@ -34,6 +34,29 @@
     end
   end
 
+  context "current and newest" do
+    let(:price_policy) { FactoryGirl.create(:instrument_price_policy) }
+    let(:overlapping_price_policy) do
+      FactoryGirl.create(
+        :instrument_price_policy,
+        start_date: (price_policy.start_date + 1.day).beginning_of_day,
+        product: price_policy.product,
+        price_group: price_policy.price_group,
+      )
+    end
+
+    before { overlapping_price_policy.update_attributes(start_date: 1.day.ago.beginning_of_day) }
```

Comment:
I'm a little confused by what's happening here in these specs. Why are we setting the `start_date` for this policy twice? And why are we checking for the validity?